### PR TITLE
added school feedback component

### PR DESCRIPTION
**Pull request by LeozMaxwellJilliams**
- added component to show school feedback which fetches performances by school token
- todo: update the component's `render` function to reuse new components

### CODE REVIEW

File: src/api/SchoolAPI.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import humps from 'humps';
+import { get, post } from './ApiUtils';
+
+export const getPerformancesByToken = (token) => {
+  return get(`school/${token}`)
+    .then((response) => humps.camelizeKeys(response));
+};
+
+export const generateToken = (data) => {
+  return post('school/token', humps.decamelizeKeys(data))
```
Comment (bmlunchbox) [inline src/api/SchoolAPI.js]: how is the token being generated?

File: src/components/feeback/SchoolFeedbackSection.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import PropTypes from 'prop-types';
+
+import { getPerformancesByToken } from '../../api/SchoolAPI';
+
+class SchoolFeedbackSection extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      performances: null
```
Comment (bmlunchbox) [inline src/components/feeback/SchoolFeedbackSection.jsx]: can you initialize an empty array / object for this ?

Comment (bmlunchbox) [inline src/components/feeback/SchoolFeedbackSection.jsx]: can you initialize an empty array for this?

**Comment by LeozMaxwellJilliams** [comment]
Right now the page just shows a table of the dance names, I'll be reusing the winners component that Peiyao and Lena make

**Comment by bmlunchbox** [comment]
nice good job

**Pull request merged**